tracing-opentelemetry/src/lib.rs

```diff
@@ -109,3 +109,10 @@ mod tracer;
 pub use span_ext::OpenTelemetrySpanExt;
 pub use subscriber::{subscriber, OpenTelemetrySubscriber};
 pub use tracer::PreSampledTracer;
+
+/// Per-span OpenTelemetry data tracked by this crate.
+#[derive(Debug, Clone)]
+pub struct OtelData {
```

Comment:
Am I correct that this needs to be public because it's used in the `PreSampledTracer` trait, but it's totally opaque to user code? Is there any reason user code might need to access or mutate this type's fields?

Reply:
Good point, fields need to be pub for althernate sdk implementations. Will fix.

Reply:
Added quick comment as well to clarify purpose a bit. Should be opaque to everyone who is not trying to implement `PreSampledTracer` directly.